spago.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: profunctor-lenses
+  publish:
+    license: MIT
+    version: 8.0.0
+    location:
+      githubOwner: purescript-contrib
+      githubRepo: purescript-profunctor-lenses
+  dependencies:
+    - arrays
```

Comment:
For a library, the spago.yaml file needs to specify version ranges and should not use a package set. For example, this should be:

```yaml
- arrays: >=7.0.0 <8.0.0
```

for parity with the bower.json file. It won't error right away but this is necessary to publish the package to the registry and for other libraries to depend on this library.

Reply:
The existing spago.dhall files don't specify anything because they're internal-only and irrelevant for publishing. The spago.yaml files, though, will be used by the registry to understand this package when publishing and are used by other libraries when pointing at this Git repo.

Reply:
> The existing spago.dhall files don't specify anything because they're internal-only and irrelevant for publishing. The spago.yaml files, though, will be used by the registry to understand this package when publishing and are used by other libraries when pointing at this Git repo.

Thanks for pointing that out, I hadn't thought about that important difference between `spago.dhall` and `spago.yaml`.

Reply:
1 sec, will do ranges for all